tests/src/FunctionalJavascript/ContactSubmissionTest.php

```diff
@@ -3,6 +3,7 @@
 namespace Drupal\Tests\webform_civicrm\FunctionalJavascript;
 
 use Drupal\Core\Url;
+use Drupal\webform_civicrm\Utils;
 
```

Comment:
Do we need the `use` here? I don't see it in `ActivitySubmissionTest.php` e.g.

Reply:
Nope, it wasn't required. I've removed it. Thanks for pointing that 👍